Frameworks/CoreText/CTFont.mm

```diff
@@ -508,19 +601,19 @@ CGFloat CTFontGetUnderlineThickness(CTFontRef font) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CGFloat CTFontGetSlantAngle(CTFontRef font) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return _DWriteFontGetSlantDegrees(font->_dwriteFontFace);
```

Comment:
It seems consistent throughout CoreText to just return a default value rather than crashing, though I'm not sure how other frameworks handle it, I'd like to conform in ours as closely as is reasonable

---
In reply to: [80785598](https://github.com/Microsoft/WinObjC/pull/1039#discussion_r80785598) [](ancestors = 80785598,80783054,80757662,80733021)